src/main/java/org/broadinstitute/hellbender/utils/dataflow/BucketUtils.java

```diff
@@ -74,4 +77,41 @@ public static OutputStream createFile(String path, PipelineOptions popts) {
             throw new UserException.CouldNotCreateOutputFile(path, x);
         }
     }
+
+    /**
+     * Copies a file. Can be used to copy e.g. from GCS to local.
+     *
+     * @param sourcePath the path to read from. If GCS, it must start with "gs://".
+     * @param popts the pipeline's options, with authentication information.
+     * @param destPath the path to copy to. If GCS, it must start with "gs://".
+     * @throws IOException
+     */
+    public static void copyFile(String sourcePath, PipelineOptions popts, String destPath) throws IOException {
+        try (
+            InputStream in = openFile(sourcePath, popts);
```

Comment:
in and fout are already in the try initializer... 
I've moved to ByteStreams.copy, thank you! That's much cleaner.